test/plugin/scenarios/solon-2.x-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,99 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+segmentItems:
+- serviceName: solon-2.x-scenario
+  segmentSize: ge 2
+  segments:
+  - segmentId: not null
+    spans:
+    - operationName: H2/JDBC/PreparedStatement/executeQuery
+      parentSpanId: 0
+      spanId: 1
+      spanLayer: Database
+      startTime: not null
+      endTime: not null
+      componentId: 32
+      isError: false
+      spanType: Exit
+      peer: not null
+      tags:
+      - {key: db.type, value: not null}
+      - {key: db.instance, value: test}
+      - {key: db.statement, value: not null}
+      skipAnalysis: 'false'
+    - operationName: '/testcase/error'
+      parentSpanId: 0
+      spanId: 2
+      spanLayer: Http
+      startTime: not null
+      endTime: not null
+      isError: false
+      componentId: not null
+      tags:
+      - {key: url, value: not null}
+      - {key: http.method, value: GET}
+      skipAnalysis: 'false'
+    - operationName: 'future/get:/testcase/error'
+      parentSpanId: 0
+      spanId: 3
+      spanLayer: not null
+      startTime: not null
+      endTime: not null
+      isError: true
+      componentId: not null
+      logs:
+      - logEvent:
+        - {key: event, value: error}
+        - {key: error.kind, value: not null}
+        - {key: message, value: not null}
+        - {key: stack, value: not null}
+      skipAnalysis: 'false'
+    - operationName: 'GET:/testcase/{test}'
+      parentSpanId: -1
+      spanId: 0
+      spanLayer: Http
+      startTime: not null
+      endTime: not null
+      componentId: 14
```

Comment:
There is no `Solon` component related. What is the meaning of your tests?

Reply:
Sorry, this isn't the final code, I just wanted to get a different computer to continue with the code